Phoniebox/rpc/PhonieboxRpcClient.py

```diff
@@ -4,17 +4,23 @@
 class PhonieboxRpcClient:
     
     def __init__(self):
-        #self.objects = objects
         self.context = None
         
-    def connect(self,addr= None):
-        if addr == None:
-            addr = "tcp://127.0.0.1:5555"
-        self.context = zmq.Context()
+    def connect(self,addr=None,zmq_context=None):
+        if zmq_context != None:
+            self.context = zmq_context
+            local_addr = "inproc://PhonieboxRpcServer"
```

Comment:
Do we want to outsource this into a config file so we can reference it in different parts of the python app?

Reply:
It is prepared for passing the address, this just defining the default